setup.Rmd

```diff
@@ -124,7 +124,7 @@ During the Rtools installation you may see a window asking you to "Select Additi
 
 ### macOS
 
-You need to install the Xcode command line tools, which requires that you [register as an Apple developer](https://developer.apple.com/programs/register/) (don't worry, it's free).
+You need to install the Xcode command line tools, which requires that you [register as an Apple developer](https://developer.apple.com/programs/enroll/) (Don't worry, it's free with an Apple ID. There is no need to pay for the $99 certification).
```

Comment:
Let's expand on this a bit.

I found this in the "Additional Information" section of https://developer.apple.com/support/enrollment/:

> Do I need to enroll to install apps on a device?
>
> No. You can install apps on a device for free with [Xcode](https://help.apple.com/xcode/mac/current/#/dev5a825a1ca). You’ll only need to enroll if you’d like to distribute apps, access beta software, and integrate with capabilities such as Siri, Apple Pay, and iCloud.

```suggestion
You need to install the Xcode command line tools, which requires that you [register as an Apple developer](https://developer.apple.com/programs/enroll/). Don't worry, this is free for an individual who only wishes to install apps, such as Xcode command line tools. Enrolling in the paid developer program is only necessary for those who want distribute apps, access beta software, and integrate with capabilities such as Siri, Apple Pay, and iCloud.
```

Reply:
Excellent!